presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/APIFileHandler.java

```diff
@@ -60,15 +59,15 @@ public Response getFile(String path)
     {
         try {
             String fileContent = AgentFileUtils.getFile(Paths.get(baseDir.toString(), path));
-            LOGGER.debug("Successfully retrieved the content of file {}", path);
+            LOGGER.debug("Successfully retrieved contents of file '%s'", path);
```

Comment:
I prefer to keep my error messages as simple as possible, and I prefer not to mix "grammar levels".
I could go for "Successfully retrieved the contents of the file at the path '%s'", but that would be pressing the verbosity a bit far, and doesn't carry any more information.